### DISCUSSION TITLE
Programmatically copying text from the Linux server to the Window's client's clipboard?

### DISCUSSION THREAD

**Discussion by cool-RR** [microsoft/terminal#17934] [Q&A] [answered]
I'm connecting from a Windows client, using Windows Terminal, to a Linux server. I would like a program that runs on that Linux server to put a piece of text in the clipboard of the Windows client. Is that possible?

**Comment by o-sdn-o** [comment, marked as answer]
Maybe you should use OSC52 sequence.

```
printf "\e]52;;$(printf "Hello world!" | base64)\a"
```

**Reply by cool-RR** [reply]
This works with some modifications:

```
printf '\e]52;c;%s\a' "$(echo -n 'Your text here' | base64)"
```

However I need this to work without outputting text to the console.

**Reply by o-sdn-o** [reply]
I corrected my version by adding `\a` at the end 😉.

**Reply by cool-RR** [reply]
Well butter my biscuits, that works.
